src/pages/MovieDetails.js

```diff
@@ -1,25 +1,63 @@
 import React, { Component } from 'react';
-
-import * as movieAPI from '../services/movieAPI';
+import PropTypes from 'prop-types';
+import { Link, Redirect } from 'react-router-dom';
 import { Loading } from '../components';
+import * as movieAPI from '../services/movieAPI';
 
 class MovieDetails extends Component {
-  render() {
-    // Change the condition to check the state
-    // if (true) return <Loading />;
+  constructor() {
+    super();
+    this.state = {
+      movie: {},
+      loading: false,
```

Comment:
Será que não faria sentido o loading começar como "true", uma vez que a página ainda não fez a requisição à API?
```suggestion
      loading: true,
```

Reply:
Muito obrigada, Bruno! Já commitei :D 🏅️🚀️